chord_drs/routes.py

```diff
@@ -93,10 +110,16 @@ def object_info(object_id):
     if not drs_object and not drs_bundle:
         return abort(404)
 
+    # Are we inside the bento singularity container? if so, provide local accessmethod
+    if request.headers.get('X-User'):
```

Comment:
I added an `X-CHORD-Internal` header to more explicitly check this
It should default to 0 (meaning not internal); if it's 1 then we're behind the auth proxy and have UNLIMITED POWER